.gitignore

```diff
@@ -1,28 +1 @@
-# Logs
```

Comment:
Could we keep this file as is?

Reply:
Well, unless these file are in the repo, there's no reason to. And clearly this file was taken from a generic .gitignore.

Reply:
Fair enough. We can update this file when it's needed then. Thanks!